client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -114,13 +115,18 @@ export const JurisdictionAdminView: React.FC = () => {
     uploadBallotManifest,
     deleteBallotManifest,
   ] = useBallotManifest(electionId, jurisdictionId)
+  const [
+    batchTallies,
+    uploadBatchTallies,
+    deleteBatchTallies,
+  ] = useBatchTallies(electionId, jurisdictionId)
```

Comment:
not sure where the right place to do it is, but somewhere you might want to condition on the audit type being `BATCH_COMPARISON`

Reply:
@jonahkagan Oh interesting, yeah. I cannot make the hook itself conditional of course, so I'd have to do that on whether I render the form or not. And then like you said earlier have it just be disabled at first.